datasets/loader_multiclass.py

```diff
@@ -204,9 +208,11 @@ def msrank(dataset_dir: Path) -> bool:
     """
     Dataset from szilard benchmarks: https://github.com/szilard/GBM-perf
 
-    TaskType:multiclass
-    NumberOfFeatures:137
-    NumberOfInstances:1.2M
+    Multiclass classification task
```

Comment:
Could you please calculate the number of classes as well? This might be done i. e. with `np.unique()`